src/api/helpers/formatWorkData.js

```diff
@@ -1,47 +1,66 @@
-/* eslint-disable */
+/*
+ * Copyright (C) 2019  Akhilesh Kumar
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
 
+/* eslint-disable */
 import _ from 'lodash';
 
 export function getWorkBasicInfo(work: object) {
 	return _.isNil(work) ? null :
 		{
 			bbid: work.bbid,
-			defaultAlias: work.defaultAlias,
+			defaultAlias: {
+				name: work.defaultAlias.name,
+				sortName: work.defaultAlias.sortName,
+				aliasLanguage: work.defaultAlias.language.name
+			},
 			disambiguation: work.disambiguation ?
 				work.disambiguation.comment : null,
 			languages: work.languageSet ?
-                work.languageSet.languages.map((language) => language.name) : null,
-            workType: work.typeId,
+				work.languageSet.languages.map((language) => language.name) : null,
+			workType: work.workType.label,
```

Comment:
Please update test-helper with inserting the workType, the test is failing at `work.workType.label`.